Tests/TurfTests/GeoJSONTests.swift

```diff
@@ -216,4 +216,32 @@ class GeoJSONTests: XCTestCase {
         
         XCTAssertEqual(decodedFeature, feature)
     }
+    
+    func testForeignMemberCoding(in object: GeoJSONObject) throws {
+        let data = try JSONEncoder().encode(object)
+        guard var json = try JSONSerialization.jsonObject(with: data, options: []) as? [String: Any?] else {
+            return
+        }
+        json["title"] = "Example Feature"
```

Comment:
Should we use a key like `__test_foreign_key` that isn't a part of any object for sure? I would also store the name of the key in a constant within this method.

Reply:
I had lifted this example from RFC&nbsp;7946, but just for fun, I changed it into a test of round-tripping a [GeoJSON-T](https://github.com/kgeographer/geojson-t/) object. The property names are defined as part of the GeoJSON-T proposal; it would be overkill to factor each one into a constant. (Ideally, there would be a Codable implementation of GeoJSON-T anyways.) I added an assertion that `when` isn’t a foreign member before setting it.